### PR TITLE
Renamings to conform .sol interface

### PR DESCRIPTION
**Pull request by taco-paco**
Change observation_timestamp -> started_at, transmission_timestamp -> updated_at to conform to conform `AggregatorV3Interface` namings

### CODE REVIEW

**Comment by archseer** [comment]
Can you rebase? I merged some changes that should make the tests pass again

**Comment by taco-paco** [comment]
> Can you rebase? I merged some changes that should make the tests pass again

done

**Comment by archseer** [comment]
Looks fine to me but it seems to have some commits from develop rebased in?

**Review by archseer — APPROVED** [review]

File: contracts/contracts/mocks/uptime_feed_mock.cairo [hunk 1]
```diff
@@ -0,0 +1,55 @@
+%lang starknet
```
Comment (krebernisak) [inline contracts/contracts/mocks/uptime_feed_mock.cairo:1]: Not sure how this file ended up her in this PR, it should be rebased out and included in the L2EP specific PR:

- https://github.com/smartcontractkit/chainlink-starknet/pull/56

File: contracts/contracts/ocr2/SequencerUptimeFeed/library.cairo [hunk 2]
```diff
@@ -0,0 +1,248 @@
+%lang starknet
```
Comment (krebernisak) [inline contracts/contracts/ocr2/SequencerUptimeFeed/library.cairo:1]: Not sure how this file ended up her in this PR, it should be rebased out and included in the L2EP specific PR:

- https://github.com/smartcontractkit/chainlink-starknet/pull/56

File: contracts/contracts/ocr2/SequencerUptimeFeed/sequencer_uptime_feed.cairo [hunk 3]
```diff
@@ -0,0 +1,87 @@
+%lang starknet
```
Comment (krebernisak) [inline contracts/contracts/ocr2/SequencerUptimeFeed/sequencer_uptime_feed.cairo:1]: Not sure how this file ended up her in this PR, it should be rebased out and included in the L2EP specific PR:

- https://github.com/smartcontractkit/chainlink-starknet/pull/56

File: contracts/contracts/utils.cairo [hunk 4]
```diff
@@ -0,0 +1,11 @@
+from starkware.cairo.common.math import assert_in_range
```
Comment (krebernisak) [inline contracts/contracts/utils.cairo:1]: Context: only used in L2EP

Not sure how this file ended up her in this PR, it should be rebased out and included in the L2EP specific PR:

- https://github.com/smartcontractkit/chainlink-starknet/pull/56

File: contracts/test/ocr2/aggregator-test.ts [hunk 5]
```diff
@@ -139,10 +139,21 @@ describe('aggregator.cairo', function () {
     console.log(config)
 
     await owner.invoke(aggregator, 'set_config', config)
+    // const receipt = await starknet.getTransactionReceipt(txHash);
+    // const decodedEvents = await aggregator.decodeEvents(receipt.events);
+    // console.log(decodedEvents)
```
Comment (krebernisak) [inline contracts/test/ocr2/aggregator-test.ts:144]: Used for local debugging? 

Let's uncomment this and instead `console.log` we assert on data like code below.

File: contracts/test/ocr2/sequencer-uptime-feed.ts [hunk 6]
```diff
@@ -0,0 +1,121 @@
+import { expect } from 'chai'
```
Comment (krebernisak) [inline contracts/test/ocr2/sequencer-uptime-feed.ts:1]: Not sure how this file ended up her in this PR, it should be rebased out and included in the L2EP specific PR:

- https://github.com/smartcontractkit/chainlink-starknet/pull/56

File: contracts/test/utils.ts [hunk 7]
```diff
@@ -0,0 +1,11 @@
+import { expect } from 'chai'
```
Comment (krebernisak) [inline contracts/test/utils.ts:1]: Not sure how this file ended up her in this PR, it should be rebased out and included in the L2EP specific PR:

- https://github.com/smartcontractkit/chainlink-starknet/pull/56

File: shell.nix [hunk 8]
```diff
@@ -18,6 +18,11 @@
     delve
     golangci-lint
     gotools
+
```
Comment (krebernisak) [inline shell.nix:21]: This was added by @archseer in another PR, what's happening here!? :)

**Review by krebernisak — CHANGES_REQUESTED** [review]
I'm not sure what happened with this PR, but it's not clean and includes a lot of changes from L2EP and changes from other branches that are either already closed or still open.

In any case, please clean this up using the following workflow:

- Rebase the renaming PR onto develop (currently on top of L2EP branch), include only the renaming - so we can quickly review and be done with this part.
- If there are specific L2EP changes you added here (not git branching/PR issues) extract those and add them to the opened L2EP PR

At that point we should be able to quickly review and merge.

Thank you!

**Comment by taco-paco** [comment]
> * If there are specific L2EP changes you added here (not git branching/PR issues) extract those and add them to the opened L2EP PR

It contains changes relevant only in context of l2ep pr that's why I made pr to l2ep branch 
I think I will just cherry-pick initial change to l2ep branch and we will continue discussion there if its required

File: contracts/contracts/mocks/uptime_feed_mock.cairo (hunk 1 above)
Comment (taco-paco) [reply contracts/contracts/mocks/uptime_feed_mock.cairo:1]: [Cherry-picked renaming commit](https://github.com/smartcontractkit/chainlink-starknet/pull/56/commits/947073583a570aa5124202725a46bd1ab07acf12) in main l2 branch

Comment (taco-paco) [reply contracts/contracts/mocks/uptime_feed_mock.cairo:1]: Closing this pr

**Pull request closed without merging**